rs/tests/boundary_nodes/bn_integration_on_playnet_test.rs

```diff
@@ -34,7 +34,7 @@ fn main() -> Result<()> {
                 .add_test(systest!(api_sync_call_test))
                 .add_test(systest!(api_canister_read_state_test))
                 .add_test(systest!(api_subnet_read_state_test))
-                .add_test(systest!(asset_canister_test))
+                .add_test(systest!(legacy_asset_canister_test))
```

Comment:
Here, we should also add the `long_asset_canister_test`. It is basically the same test as `bn_integration_test`, but it uses real certificates.

Reply:
Done.